subprojects/core-api/src/main/java/org/gradle/api/artifacts/Configuration.java

```diff
@@ -295,16 +297,24 @@ public String determineName(Configuration c) {
     /**
      * Gets the set of dependencies directly contained in this configuration
      * (ignoring superconfigurations).
+     * <p>
+     * This method does not resolve the configuration. Therefore, the return values does not include
```

Comment:
❌ I'd rephrase this and use the term "declared" dependencies here to make the distinction from "resolved" more obvious. The objects here are completely different from the ones you get in the resolution result. Depending on how the Configuration is resolved, they can end up pointing to something completely different, e.g. because of substitutions.

Reply:
done